frontend/views/containers/user-settings/UserProfile.vue

```diff
@@ -2,25 +2,10 @@
   .settings-container
     span.c-username @{{ ourUsername }}
 
-    form(@submit.prevent='')
-      .c-avatar
-        label.c-avatar-field
-          avatar.c-avatar-img(
-            :src='attributes.picture'
-            ref='picture'
-          )
-          i18n.link.c-avatar-text Change avatar
-
-          input.sr-only(
-            type='file'
-            name='profilePicture'
-            accept='image/*'
-            @change='fileChange($event.target.files)'
-            placeholder='http://'
-            data-test='profilePicture'
-          )
-        // TODO #658
-        banner-scoped.c-pictureMsg(ref='pictureMsg' data-test='pictureMsg')
+    avatar-upload(
+      :avatar='$store.getters.ourUserIdentityContract.attributes.picture'
```

Comment:
Instead of `$store.getters.ourUserIdentityContract.attributes.picture`, which might result in an exception because `.attributes` could return undefined, use `attributes.picture` (there's already a computed property in this Vue component that will return an empty object if `.attributes` is undefined)